api/.gitignore

```diff
@@ -1,2 +1,4 @@
 /node_modules
 /dist
+.idea/*
```

Comment:
What is .idea?

Reply:
It's just a hidden folder that's added when you work with some of Jetbrains products. I didn't want to add push it to GitHub, so I added it to gitignore.

Reply:
Oh, that makes a lot of sense